src/components/MAPI/clusters/ClusterDetail/ClusterDetailWidgetCreated.tsx

```diff
@@ -10,6 +10,12 @@ const StyledDot = styled(Dot)`
   padding: 0;
 `;
 
+const CapitalizedText = styled.div`
+  &:first-letter {
+    text-transform: uppercase;
+  }
+`;
```

Comment:
Could we just style the `<Date>` component here? Just a small nitpick, but I was wondering if we could avoid another wrapper 😅